src/lightbox/download.js

```diff
@@ -73,7 +73,4 @@ export const downloadFileToCache = async (tempUrl: string, fileName: string): Pr
   RNFetchBlob.config({
     path: `${RNFetchBlob.fs.dirs.CacheDir}/${fileName}`,
     fileCache: true,
-    useDownloadManager: true,
```

Comment:
```
These property are technically the sub properties of the property
`addAndroidDownloads`.
```

A quick link to the documentation would help someone in the future verify this. 🙂

```
Normally rnfb will use its internal implementation for downloading
based on the `OkHttp` android library.
```

Could you say more in the commit message, ideally with a link to documentation, about why this is relevant to the change?

Reply:
Also, I think a link to my report on CZO might be helpful in the commit message. It has some additional context about how the error was discovered; specifically, that I was working on `exact_by_default`. (That context isn't necessary to justify fixing the error, so I don't think it needs to be explained in the commit message. But a link to the discussion might help satisfy people's curiosity about how it was discovered.)

Reply:
done.